parser.js

```diff
@@ -1,8 +1,7 @@
-'use strict';
-var ok = { type: 'ok' };
-var unknown = { type: 'unknown' };
-var runTogether = { type: 'ok', runTogether: true };
-var lineBreak = { type: 'line-break' };
+var ok = Object.freeze({ type: 'aspellOk' });
+var unknown = Object.freeze({ type: 'aspellUnknown' });
+var runTogether = Object.freeze({ type: 'aspellOk', runTogether: true });
+var lineBreak = Object.freeze({ type: 'aspellLineBreak' });
```

Comment:
Could you give some rationale for changing the names? Also, why freeze this?

Reply:
@rexxars 

1. Why did I freeze the object? For two reasons, in large projects someone may eventually try to change that object, so every event can have a new name and cause unexpected problems.

2. Why a new name? This EventEmitter can be reused for more functionality from other libraries. This way there is less probability of conflict.

Reply:
I don't really like the idea of prefixing event names here - you shouldn't use the same event emitter for non-aspell things. You don't do `request.on('requestEnd')`, you just do `request.on('end')` - I'd like to stay in line with the established conventions for node.